website/templates/prereg_landing_page.mako

```diff
@@ -23,16 +23,10 @@
 
 <%def name="existingPrereg(size=None)">
 <% size = size or '' %>
-<div id="existingPrereg${size}" class="prereg-exisitng-prereg p-md osf-box box-round clearfix m-b-lg" style="display:none">
+<div id="existingPrereg${size}" class="prereg-exisitng-prereg p-md osf-box box-round clearfix m-b-lg" style="display:none; style="width: 100%;"">
```

Comment:
This may be outside the scope of this PR, but this usage of a mako variable in a CSS ID is smelly. Relying on the suffix of a CSS ID is a bad idea.

Also, there's a typo in `prereg-existing-prereg`.